src/HiveSpace.UserService/README.md

```
@@ -0,0 +1,730 @@
+- .NET 8.0 SDK
+- SQL Server (localhost:1433 by default)
+- Visual Studio 2022 or VS Code with C# extension
+
+### Running the Service
+```bash
+# From the API project directory
+cd src/HiveSpace.UserService/HiveSpace.UserService.Api
+dotnet run
+```
+
+**Service URL**: https://localhost:5001  
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)